tests/build-cache-integration.sh

```
@@ -0,0 +1,105 @@
+  args=(--builder "$builder" --progress=plain
+    --cache-to "type=registry,ref=$BUILD_CACHE_REPOSITORY:project-target-platform-main,mode=max,image-manifest=true")
+  if [ "$n" = 2 ]; then
+    args+=(--cache-from "type=registry,ref=$BUILD_CACHE_REPOSITORY:project-target-platform-main")
+  fi
+  if ! docker buildx build "${args[@]}" "$tmp/context" > "$tmp/build-$n.log" 2>&1; then
+    tail -35 "$tmp/build-$n.log" >&2
+    exit 1
+  fi
+done
+[ -s "$tmp/volume-1" ] && [ -s "$tmp/volume-2" ]
+! cmp -s "$tmp/volume-1" "$tmp/volume-2"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)